draco/analysis/ringmapmaker.py

```diff
@@ -202,7 +202,7 @@ def process(self, gstream):
         el = self.span * np.linspace(-1.0, 1.0, self.npix)
 
         # Create empty ring map
-        hv = containers.HybridVisStream(el=el, axes_from=gstream, attrs_from=gstream)
+        hv = containers.HybridVisStream(el=el, axes_from=gstream)
```

Comment:
why was it necessary to remove the attributes copy?

Reply:
As it was before the `el` axis of `containers.HybridVisStream` would appear labelled as `ns`, which is the axis corresponding to NS baselines in the input container and that is converted to (sin) angles in the declination dimension (`el`). I think this suggests there may be a lower level bug in the way containers are created.